frameworks/hdfs/universe/config.json

```diff
@@ -14,6 +14,11 @@
           "description":"The principal for the HDFS service instance.",
           "type":"string",
           "default":"hdfs-principal"
+        },
+        "cmd_prefix" : {
```

Comment:
Feels a little iffy putting this option front and center in the `service` section but there currently isn't an obviously better place to put it. If more options like this are added we may want to break them out into an `advanced` section at some point but that's more of a UI/product thing.